Converters/Rhino/Speckle.Converters.RhinoShared/ToSpeckle/Raw/NurbsCurveConverter.cs

```diff
@@ -58,7 +58,7 @@ public SOG.Curve Convert(RG.NurbsCurve target)
     }
     else
     {
-      // POC: report CONVERTED WITH WARNING
+      throw new SpeckleConversionException($"Failed to extract polyline from {target}");
```

Comment:
I guess we're preventing any nurbs curve without a display value to be sent with the new required keyword - seems like a good change, but we should revisit implications of  `required` on displayvalues in our classes in general post merge

Reply:
yep, we can continue the discussion, I've updated this part of the code a little, since there was another throw that was needed.